src/Compilers/CSharp/Portable/Symbols/Source/SourceConstructorSymbol.cs

```diff
@@ -164,10 +162,10 @@ internal override OneOrMany<SyntaxList<AttributeListSyntax>> GetAttributeDeclara
             return OneOrMany.Create(((ConstructorDeclarationSyntax)this.SyntaxNode).AttributeLists);
         }
 
-        internal override bool IsNullableAnalysisEnabled()
+        protected override bool IsNullableAnalysisEnabled(bool isNullableAnalysisEnabledFlag)
```

Comment:
>IsNullableAnalysisEnabled [](http://example.com/codeflow?start=32&length=25)

I do not see much advantage in introducing this weird, round about way of special casing constructors. I prefer to keep the original approach where we override ```IsNullableAnalysisEnabled``` here and check ```flags``` directly. Instead of making ```flags``` private, we can simply rename the field to something like "flagsDoNotUseDirectly", so that people would be very cautious of introducing new usage points for the field.

Reply:
Also, #68347 introduces another "valid" use of ```flags``` in a derived type.